### ISSUE TITLE
Hi Johan, project has huge potential!

### ISSUE THREAD

**Issue by Fenny** [johanholmerin/browsercast#1]
Hey Johan,

I'm impressed with what you did here, you are the only one providing an actual in-browser local cast player without extentions.
Should we make this code compatible with firefox, is this possible?

Let me know!

- Fenny

**Comment by johanholmerin** [comment]
Thanks.
I don't see any point in doing that, since only Chrome supports Chromecast.

**Comment by Fenny** [comment]
Thanks for your response, I didnt knew that.
Firefoz mobile supports casting, not sure if it's the same.

Do you think your library can support the file object from https://github.com/webtorrent/webtorrent

So we could stream torrents to the chromecast in the browser?

**Comment by johanholmerin** [comment]
Firefox for Android supports casting video elements, but does not support the cast framework. Once Firefox implements the Presentation API it should be possible to add support.

I have tried running WebTorrent on the Chromecast itself, but it’s so low powered that it keeps crashing. It is however possible to add WebTorrent to the sender and stream to the receiver. I have created a branch with a rough implementation.
